dt-core/admin/menu/tabs/tab-custom-lists.php

```diff
@@ -261,17 +440,18 @@ public function channels_box(){
                 </thead>
                 <tbody>
                     <?php foreach ( $fields as $channel_key => $channel_option ) :
-                        if ( $channel_option['type'] !== 'communication_channel' ){
+                        if ( $channel_option['type'] !== 'communication_channel' ) {
                             continue;
                         }
 
                         $enabled = !isset( $channel_option['enabled'] ) || $channel_option['enabled'] !== false;
                         $hide_domain = isset( $channel_option['hide_domain'] ) && $channel_option['hide_domain'] == true;
-                        if ( $channel_key == 'phone' || $channel_key == 'email' || $channel_key == 'address' ){
+                        if ( $channel_key == 'contact_phone' || $channel_key == 'contact_email' || $channel_key == 'contact_address' ){
```

Comment:
This removes them from the Communication Channels list?

Reply:
I believe this was previously the case, although; the key reference was wrong. Are phone/email/address to be included by default?